flake8-requirements.txt

```
@@ -1,27 +1,27 @@
-flake8~=3.8.3
+flake8==3.8.3
 
 # Plugins
 # Ref: https://github.com/DmytroLitvinov/awesome-flake8-extensions
 
-flake8-bandit~=2.1.2                    # runs bandit
-flake8-black~=0.2.1                     # runs black
-flake8-broken-line~=0.2.1               # forbey "\" linebreaks
-flake8-builtins~=1.5.3                  # builtin shadowing checks
-flake8-coding~=1.3.2                    # coding magic-comment detectiong
-flake8-comprehensions~=3.2.3            # comprehension checks
-flake8-deprecated~=1.3                  # deprecated call checks
-flake8-docstrings~=1.5.0                # pydocstyle support
-flake8-executable~=2.0.4                # shebangs
-flake8-fixme~=1.1.1                     # "fix me" counter
-flake8-functions~=0.0.4                 # function linting
-flake8-html~=0.4.1                      # html output
-flake8-if-statements~=0.1.0             # condition linting
+flake8-bandit==2.1.2                    # runs bandit
```

Don't freeze bandit please, we want this to always pull in the newest version regardless of whether dependabot has updated it or not.

```suggestion
flake8-bandit~=2.1.2                    # runs bandit
```